CHANGES.md

```diff
@@ -6,6 +6,7 @@ The following changes are not yet released, but are code complete:
 
 Features:
  - Introduces a new upload type for Dockets from ACMS and enhances code maintainability by refactoring common upload logic into reusable functions.([#368](https://github.com/freelawproject/recap-chrome/pull/371))
+ - Updates the AppellateDelegate class to upload Docket reports from ACMS
```

Comment:
This should be written for humans, not coders. Perhaps: 

```suggestion
 - Docket reports from ACMS are now uploaded to CourtListener.
```

Reply:
got it! I'll update the chagelog.